.github/scripts/generate_ci_workflows.py

```diff
@@ -824,8 +824,8 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
     DockerWorkflow(
         build_environment="docker-builds",
         docker_images=sorted(DOCKER_IMAGES),
-        # Run weekly to ensure they can build
-        is_scheduled="1 * */7 * *",
+        # Run every Wednesday to ensure they can build
+        is_scheduled="1 * * * 3",
```

Comment:
If this works we should probably set a specific hour since this runs every hour on the first minute I feel